src/main/java/com/raeic/embarker/reflections/util/ReflectionManager.java

```diff
@@ -50,13 +50,23 @@ public WorldServerReflection getWorldServerReflection() {
     }
 
     public Class<?> getNMSClass(String name) throws ClassNotFoundException {
-        return Class.forName("net.minecraft.server."
-                             + Bukkit.getServer().getClass().getPackage().getName().split("\\.")[3] + "." + name);
+        // Check if we can detect a Minecraft version
+        String[] packageName = Bukkit.getServer().getClass().getPackage().getName().split("\\.");
```

Comment:
Can we extract this into a shared spot and only do this once when the plugin is enabled?

Reply:
oke